rm/rm-node/build.gradle

```diff
@@ -20,10 +20,10 @@ dependencies {
     runtime "org.objectweb.proactive:programming-extension-pamr:${programmingVersion}"
 
     // script engines
-    runtime 'org.jruby:jruby-complete:9.0.5.0'
-    runtime 'org.python:jython-standalone:2.7.0'
+    runtime 'org.jruby:jruby-complete:9.1.12.0'
+    runtime 'org.python:jython-standalone:2.7.1'
```

Comment:
The jython version 2.7.1. seems to refer to the 2.7.1b3 which is beta3. 
I think using a beta version is too risky, because jython is extensively used.